### ISSUE TITLE
RateIt extension doesn't work

### ISSUE THREAD

**Issue by 2plus** [bolt/bolt#2709]
Hi, I installed the extension, and I added to the default template index.twig `{{ rateit() }}`

* The first problem with the display of the extension, when I want to have several of them on one page.
![](http://i.imgur.com/OhtU5a9.png)

* The second problem is that extension does not work. **After the page refreshes, no data are saved.**

Please help me, maybe I'm doing something wrong.

**Comment by 2plus** [comment]
http://185.49.14.123/bolt/

**Comment by GawainLynch** [comment]
> The first problem with the display of the extension, when I want to have several of them on one page

The extension doesn't support more than one per page. I know that @tobias2k made some changes for me during the Bolt 1.x period, but I don't know if he was able to get that working (and then I broke it with porting to Bolt 2.

> After the page refreshes, no data are saved.

Did you update your database after install?  If not, visit http://185.49.14.123/bolt//dbcheck and that will prompt you.

If that still doesn't work, just let me know :grin:

**Comment by 2plus** [comment]
Yes, I update database.

http://185.49.14.123/bolt/bolt
login: admin
pass: 123456

**Comment by GawainLynch** [comment]
No problems.  About to head out for a dinner with friends, but shall have a look in the morning (UTC).

**Comment by 2plus** [comment]
@GawainLynch Ok, Thanks

**Comment by GawainLynch** [comment]
What I am seeing in the browser developer tools network console is a HTTP 500 error on the async route.

Are you able to tell me what is ending up in the error logs?  Alternatively if that is a totally standalone server installation, could I get console access?  if that is possible, email me at gawain.lynch@gmail.com , and I can give you my public SSH key.

**Comment by GawainLynch** [comment]
OK, I think I can see now what is going on...  
```
POST http://185.49.14.123/async/RateIt/ 500 (Internal Server Error)
```
Your site root is `http://185.49.14.123/bolt` so the request should have been for `http://185.49.14.123/bolt/async/RateIt/`.

I have pushed a fix to RateIt and tagged 2.0.1, updated your install to the latest and it works for me :+1:

**Comment by 2plus** [comment]
Thanks you very much :)
